diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentNoKDocExpected.kt

```diff
@@ -22,9 +22,8 @@ class A constructor(
 ){}
 
 /**
- * @property name
-  * @property lastName
-*/
+ * @property lastName
```

Comment:
Please add an example with two or more accessible properties to test that several tags are added correctly.